src/main/java/uk/gov/ida/rp/testrp/controllogic/AuthnRequestSenderHandler.java

```diff
@@ -22,7 +22,7 @@
 import java.util.Optional;
 import java.util.function.Function;
 
-import static java.text.MessageFormat.format;
+import static java.lang.String.format;
```

Comment:
MessageFormat uses curly braces for the interpolation pattern which clashes with the environment variable override syntax that the Dropwizard YAML parser uses. String.format uses % signs which plays nicer.